CHANGELOG.asciidoc

```diff
@@ -3,6 +3,11 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.17.20]]
+=== Beats version 7.17.20
+https://github.com/elastic/beats/compare/v7.17.19\...v7.17.20[View commits]
+
```

Comment:
```suggestion

==== Added

*Affecting all Beats*

- Upgrade Go version to 1.21.9. {pull}38728[38728]
```

Missed this in https://github.com/elastic/beats/pull/38728

Reply:
You're right, I missed that in BC2 we got this one in (that wasn't the case in BC1).